app/components/primer/hidden_text_expander.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Primer
+  # Use HiddenTextExpander to indicate and toggle hidden text.
+  class HiddenTextExpander < Primer::Component
+    # @example Default
+    #   <%= render(Primer::HiddenTextExpander.new) %>
+    #
+    # @example Inline
+    #   <%= render(Primer::HiddenTextExpander.new(inline: true)) %>
+    #
+    # @example Styling the button
+    #   <%= render(Primer::HiddenTextExpander.new(button_arguments: { p: 1, classes: "my-custom-class" })) %>
```

Comment:
Would it be possible for the examples to actually toggle hidden text?

Reply:
not now, it requires some JS behaviors that we can explore adding in the future

Reply:
🤔 gotcha. I think we should add JS behaviors to the Beta criteria then 😄

Reply:
yeah, but I don't think this exact component should have the behavior, it is part of a bigger component problably